src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/resource/LocationsController.kt

```diff
@@ -0,0 +1,71 @@
+package uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.resource
+
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.Parameter
+import io.swagger.v3.oas.annotations.media.ArraySchema
+import io.swagger.v3.oas.annotations.media.Content
+import io.swagger.v3.oas.annotations.media.Schema
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import io.swagger.v3.oas.annotations.responses.ApiResponses
+import io.swagger.v3.oas.annotations.tags.Tag
+import org.springframework.http.MediaType
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.config.ErrorResponse
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.model.Location
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.service.LocationsService
+
+@Tag(name = "Locations Controller")
+@RestController
+@RequestMapping(value = ["locations"], produces = [MediaType.APPLICATION_JSON_VALUE])
+class LocationsController(private val locationsService: LocationsService) {
+
+  @Operation(summary = "Endpoint to return a list of locations for a prison")
+  @ApiResponses(
```

Comment:
List of appointment type / video type locations in a prison?

Reply:
Already done this locally so will be in next PR.